modules/kueue-upgrading-1-1.adoc

```diff
@@ -0,0 +1,40 @@
+// Module included in the following assemblies:
+//
+// * ai_workloads/kueue/install-disconnected.adoc
+// * ai_workloads/kueue/install-kueue.adoc
+
+:_mod-docs-content-type: PROCEDURE
```

Comment:
Should we use a version number so this doc could be general (n -1 -> n upgrade?)

Or we continue to update this as we add new releases. We can add a page on how the upgrade should work?

Reply:
I wondered this too - I figured this was a one off if there will be seamless upgrades later. Is the plan to have a similar procedure for 1.2? Do we know when seamless upgrades are happening?